include/SubWindow.h

```diff
@@ -43,16 +44,44 @@ class QWidget;
 class EXPORT SubWindow : public QMdiSubWindow
 {
 	Q_OBJECT
+	Q_PROPERTY(QBrush activeColor READ activeColor WRITE setActiveColor)
+	Q_PROPERTY(QColor textShadowColor READ textShadowColor WRITE setTextShadowColor)
+	Q_PROPERTY(QColor borderColor READ borderColor WRITE setBorderColor)
+
 public:
-	SubWindow(QWidget *parent=NULL, Qt::WindowFlags windowFlags=0);
+	SubWindow(QWidget *parent = NULL, Qt::WindowFlags windowFlags = 0);
 	// same as QWidet::normalGeometry, but works properly under X11 (see https://bugreports.qt.io/browse/QTBUG-256)
 	QRect getTrueNormalGeometry() const;
+	QBrush activeColor() const;
+	QColor textShadowColor() const;
+	QColor borderColor() const;
+	void setActiveColor(const QBrush & b);
+	void setTextShadowColor(const QColor &c);
+	void setBorderColor(const QColor &c);
+
 protected:
 	// hook the QWidget move/resize events to update the tracked geometry
-	virtual void moveEvent(QMoveEvent * event);
-	virtual void resizeEvent(QResizeEvent * event);
+	virtual void moveEvent( QMoveEvent * event );
+	virtual void resizeEvent( QResizeEvent * event );
+	virtual void paintEvent( QPaintEvent * pe );
+	
 private:
+	const QSize m_buttonSize;
+	const int m_titleBarHeight = 24;
```

Comment:
Appearently this is failing the Travis builds, as its not allowed by the C++ ISO standard: 
http://stackoverflow.com/questions/12757501/why-the-iso-c-standard-forbids-the-initialization-for-members